recipe/pyqtwebkit/QtWebKit/QtWebKit.toml

```diff
@@ -0,0 +1,6 @@
+# Automatically generated configuration for PyQt5.QtWebKit.
```

Comment:
Are these files are being generated when a build is being ran? There are some `so` files around, as well other resources that are not referenced from the `meta.yaml`

Reply:
Woops, old cruft didn't mean to commit that. Fixed in https://github.com/conda-forge/pyqt-feedstock/pull/117/commits/736554c64e1d028a6745edb2fb901b0ede259521, thanks.